boards/scdz/ch32v307evt/ch32v307evt.yaml

```
@@ -0,0 +1,12 @@
+identifier: scdz_ch32v307evt
```

```suggestion
identifier: ch32v307evt
```
this should fix the CI error.